### DISCUSSION TITLE
Converting a tailwind template to daisy automatically

### DISCUSSION THREAD

**Discussion by drpark** [saadeghi/daisyui#2399] [Ideas / feature requests]
Hi, is there anyking of pre-post process or or simple script to convert tailwind classes in html to daisyui ? This would help to reduce html size of former website with a load of pages and switching to daisy ui easiest

Thanks for answers

**Comment by saadeghi** [comment]
As far as I know there's no such thing.  
The challenge is, a page designed with Tailwind CSS is Judd for us if using utility classes and there's no context about the UI. A script can't guess when a `<a>` tag is supposed to be styled as a tab, or button or inline link.  
And any of these elements could use very different utility classes.

It may be possible to make a tool to do this using AI however.
